src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,149 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
+
+  @include small {
+    height: 216.27px;
+  }
+
+  @include medium-and-large {
+    height: 433px;
+  }
+
+  @include extra-large {
+    height: 519.5px;
+  }
+}
+
+$background-block-height: ($vertical-overlap *  4);
```

Comment:
I wouldn't use the `$vertical-overlap` var here if its not concerning itself with any vertical overlap.